README.md

```diff
@@ -10,76 +10,34 @@ Active Admin is a Ruby on Rails framework for creating elegant backends for webs
 
 ## State of the project
 
-### 1.0.0
+1.0.0.pre5 is the last release to support Rails 3.2 and Ruby 1.9.3.
```

Comment:
Given that the new `inherited resources` realease supports ruby >= 2.1 only, I'd say we need to drop 2.0 here as well.